engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql

```diff
@@ -0,0 +1,53 @@
+ALTER VIEW `cloud`.`service_offering_view` AS
```

Comment:
compilation fails because of missing license header. @pbankonier please add it

Reply:
That's right. We're going to add the header